runtime/common/src/lib.rs

```diff
@@ -483,3 +483,56 @@ pub mod xcm_transactor {
 		}
 	}
 }
+
+pub mod foreign_investments {
+	use cfg_primitives::Balance;
+	use cfg_traits::SimpleCurrencyConversion;
+	use cfg_types::{
+		fixed_point::{FixedPointNumberExtension, Rate},
+		tokens::CurrencyId,
+	};
+	use frame_support::pallet_prelude::PhantomData;
+	use sp_runtime::{traits::Get, DispatchError};
+
+	/// Simple stable coin amount converter from one stable to another. Provides
+	/// a synchronous estimation of the amount of one stable currency in
+	/// another, e.g., the worth of 100 EthWrappedDai in USDC.
+	///
+	/// For now, converts any `ForeignAsset` into another with the configured
+	/// rate.
+	///
+	/// NOTE: Should be deprecated ASAP!
```

Comment:
How do we want to depcreated it?

Reply:
I thought that at some point either an oracle or TWAMMs could return an exchange estimation instead of this way too simple single rate based conversion?!

Reply:
I was not sure as we discussed introducing a simple conversion rate during the MultiCurrency call last week. I thought that due to potential fees and some practical trading inefficiencies, having rates of `0.99...` are more realistic. WDYT @offerijns ?

Reply:
On a note, I have set the conversion rate to exactly `Rate::one()` for now